presto-phoenix/src/main/java/io/prestosql/plugin/phoenix/PhoenixClient.java

```diff
@@ -156,7 +156,7 @@ public void execute(ConnectorSession session, String statement)
             while (resultSet.next()) {
                 String schemaName = resultSet.getString("TABLE_SCHEM");
                 // skip internal schemas
-                if (schemaName != null && !schemaName.equalsIgnoreCase("information_schema")) {
+                if (filterSchema(schemaName)) {
```

Comment:
do we still need `listSchemas` here?

Reply:
I think so. 
https://github.com/prestosql/presto/blob/b52bcd71d4a3fc17fcb74eb3aa1bd291b4111128/presto-phoenix/src/main/java/io/prestosql/plugin/phoenix/PhoenixClient.java#L156